ui/appui-react/src/appui-react/statusfields/SelectionCount.scss

```diff
@@ -6,9 +6,18 @@
   gap: var(--iui-size-2xs);
   display: flex;
   align-items: center;
-  padding: 0 var(--iui-size-2xs) 0 var(--iui-size-2xs);
+  min-width: var(--iui-size-2xl);
+  padding-left: var(--iui-size-2xs);
 
   .icon {
     color: var(--iui-color-icon);
   }
+
+  &.left-statusBar-corner {
```

Comment:
`uifw-` prefixes still needed. Fine to simplify, since it's a nested element i.e.: `.uifw-left-corner`

Reply:
Updated.